src/navigation/_ApplicationSwitcher.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ContentContainer from 'terra-content-container';
+import ActionHeader from 'terra-action-header';
+import List, { Item } from 'terra-list';
+import Hyperlink from 'terra-hyperlink';
+import { disclosureManagerShape, withDisclosureManager } from 'terra-application/lib/disclosure-manager';
+import classNames from 'classnames/bind';
+import styles from './ApplicationSwitcher.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Apps config
+   */
+  apps: PropTypes.arrayOf(
+    PropTypes.shape({
```

Comment:
are these values straight from the apps configuration provided through the site config? If so, do we need to update the base path / add a base path option to these applications or are the url values the base paths in this case?

Reply:
only url and site are used here, base path isn't so i didn't define it.